File: src/scaffolds/backends/dotnet/Models/OperationLog.cs
namespace DotnetMysqlBackend.Models
{
    /// <summary>
    /// 操作日志实体
    /// </summary>
    public class OperationLog
    {
        public long Id { get; set; }
        public string? Username { get; set; }
        public string? Operation { get; set; }
        public string? Method { get; set; }
        public string? Params { get; set; }
        public long? ExecuteTime { get; set; }
        public string? Ip { get; set; }
        public DateTime? CreateTime { get; set; }
    }
}

File: src/scaffolds/demos/booking/backend/src/main/java/com/example/entity/OperationLog.java
package com.example.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;
import java.time.LocalDateTime;

/**
 * 操作日志实体类
 */
@Data
@TableName("operation_log")
public class OperationLog {
    /**
     * 主键ID
     */
    @TableId(type = IdType.AUTO)
    private Long id;
    
    /**
     * 操作用户名
     */
    private String username;
    
    /**
     * 操作描述
     */
    private String operation;
    
    /**
     * 方法名
     */
    private String method;
    
    /**
     * 请求参数
     */
    private String params;
    
    /**
     * 执行时长（毫秒）
     */
    private Long executeTime;
    
    /**
     * IP地址
     */
    private String ip;
    
    /**
     * 创建时间
     */
    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createTime;
}
